lib/pages/caregiver/plan_details_page.dart

```diff
@@ -0,0 +1,197 @@
+import 'dart:developer';
+
+import 'package:flutter/material.dart';
+import 'package:fokus/model/button_type.dart';
+import 'package:fokus/utils/app_locales.dart';
+import 'package:fokus/utils/theme_config.dart';
+import 'package:fokus/widgets/app_header.dart';
+import 'package:fokus/widgets/chips/attribute_chip.dart';
+import 'package:fokus/widgets/dialogs/dialog.dart';
+import 'package:fokus/widgets/item_card.dart';
+import 'package:fokus/widgets/popup_menu_list.dart';
+import 'package:fokus/widgets/segment.dart';
+
+class CaregiverPlanDetailsPage extends StatefulWidget {
+  @override
+  _CaregiverPlanDetailsPageState createState() =>
+      new _CaregiverPlanDetailsPageState();
+}
+
+class _CaregiverPlanDetailsPageState extends State<CaregiverPlanDetailsPage> {
```

Comment:
This page is very similar to the child plan details. We might consider merging them into one and parameterizing the differences. It's probably best to wait with that until the logic is added though.